CreeDictionary/API/models.py

```diff
@@ -196,10 +201,19 @@ class Meta:
             models.Index(fields=["analysis"]),
             # text index benefits fast wordform matching (see search.py)
             models.Index(fields=["text"]),
+            # When we *just* want to lookup text wordforms that are "lemmas"
+            # (Note: Eddie thinks "head words" may also be lumped in as "lemmas")
+            # Used by:
+            #  - affix tree intialization
+            #  - sitemap generation
+            models.Index(fields=["is_lemma", "text"], name="lemma_text_idx"),
+            # TODO: do we need these indices?
```

Comment:
grep for `pos=` or `inflectional_category=` and you’ll find they’re an attempt to speed up [building the preverb cache](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/ed8a43b138ab03d45a21800940a7fec23c89376a/CreeDictionary/API/search/lookup.py#L157).